index.html

```diff
@@ -46,127 +42,75 @@ <h1>Cakes for any event<br />
 											<h3>Cupcakes</h3>
 											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
 											<ul class="actions">
-												<li><a href="orders.html" class="button prefill-button" data-order-type="cupcakes">Order This</a></li>
+												<li><a href="menu/cupcakemenu.html" class="button">Learn More</a></li>
 											</ul>
 										</article>
 										<article>
-											<a href="#" class="image"><img src="images/cakes/cake_pic02.jpg" alt="thanksgiving pie" /></a>
-											<h3>Holidays</h3>
+											<a href="menu/cookiesmenu.html" class="image"><img src="images/pic02.jpg" alt="placeholder image" /></a>
+											<h3>Cookies</h3>
 											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
 											<ul class="actions">
-												<li><a href="orders.html" class="button prefill-button" data-order-type="holiday">Order This</a></li>
+												<li><a href="menu/cookiesmenu.html" class="button">Learn More</a></li>
 											</ul>
 										</article>
 										<article>
-											<a href="#" class="image"><img src="images/cakes/cake_pic14.jpg" alt="company anniversary cake" /></a>
-											<h3>Corporate Events</h3>
+											<a href="menu/piesmenu.html" class="image"><img src="images/pic02.jpg" alt="placeholder image" /></a>
+											<h3>Pies</h3>
 											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
 											<ul class="actions">
-												<li><a href="orders.html" class="button prefill-button" data-order-type="corporate-event">Order This</a></li>
+												<li><a href="menu/piesmenu.html" class="button">Learn More</a></li>
 											</ul>
 										</article>
 										<article>
-											<a href="#" class="image"><img src="images/cakes/cake_pic13.jpg" alt="birthday cake" /></a>
-											<h3>Birthdays</h3>
+											<a href="tasting.html" class="image"><img src="images/cakes/cake_pic01.jpg" alt="wedding cake" /></a>
+											<h3>Weddings</h3>
 											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
 											<ul class="actions">
-												<li><a href="orders.html" class="button prefill-button" data-order-type="birthday">Order This</a></li>
+												<li><a href="tasting.html" class="button primary">Schedule Tasting</a></li>
+												<li><a href="options.html" class="button">See Options</a></li>
 											</ul>
 										</article>
 									</div>
-									<footer id="footer" class="align-center cta">
+									<!-- <footer id="footer" class="align-center cta">
```

Comment:
I commented this out for now. I didn't delete for now in case we want to have the option for custom ordering later. I am also okay removing this for now to keep things clean.